cypress/pageobject/Sample/SampleTestCreate.ts

```diff
@@ -0,0 +1,79 @@
+export class SampleTestPage {
+  sampleTestReportHistory = 0;
+  patientName = ""; // to search for patient in sample page
+
+  visitPatientPage() {
+    cy.awaitUrl("/patients");
+  }
+
+  visitPatientDashboardPage() {
```

Comment:
```suggestion
  visitPatientDashboardPage() {
```

- always try to use proper id's and not class name
- remove all unwanted function as a lot of changes requested